test/integration/server-discovery-and-monitoring/server_discover_and_monitoring.test.ts

```diff
@@ -0,0 +1,10 @@
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+describe('SDAM Unified Tests (Node Driver)', function () {
+  // TODO(NODE-5723): Remove this once the actual unified tests (test/spec/server-disovery-and-monitoring/logging) are passing
```

Comment:
Can you summarize what is different about the tests from the specs here? Would it just be "logs for TopologyDescriptionChangedEvent are omitted"?